src/main/java/seedu/dukeofbooks/command/CommandResult.java

```diff
@@ -5,26 +5,26 @@
 import seedu.dukeofbooks.data.book.Book;
 
 public class CommandResult {
-        /** The feedback message to be shown to the user. Contains a description of the execution result */
-        public final String feedbackToUser;
+    /** The feedback message to be shown to the user. Contains a description of the execution result */
+    public final String feedbackToUser;
 
-        /** The list of book that was produced by the command */
-        private final List<Book> relevantBook;
-    
-        public CommandResult(String feedbackToUser) {
-            this.feedbackToUser = feedbackToUser;
-            relevantBook = null;
-        }
-    
-        public CommandResult(String feedbackToUser, List<Book> relevantBook) {
-            this.feedbackToUser = feedbackToUser;
-            this.relevantBook = relevantBook;
-        }
-    
-        /**
-         * Returns a list of Book that was produced by the command, if any.
-         */
-        public List<Book> getRelevantBook() {
-            return relevantBook;
-        }
+    /** The list of book that was produced by the command */
+    private final List<Book> relevantBook;
+
+    public CommandResult(String feedbackToUser) {
+        this.feedbackToUser = feedbackToUser;
+        relevantBook = null;
+    }
+
+    public CommandResult(String feedbackToUser, List<Book> relevantBook) {
+        this.feedbackToUser = feedbackToUser;
+        this.relevantBook = relevantBook;
+    }
```

Comment:
Suggesting to give some documentation on this so that there is context on why overloaded method is needed.

It seems like giving a feedback without a book will not happen?